FE/src/components/interviewSettingPage/QuestionSelectionBox/QuestionTabPanelItem.tsx

```diff
@@ -91,15 +91,23 @@ const TabPanelItem: React.FC<TabPanelItemProps> = ({
             border-radius: 0 0 1rem 0;
           `}
         >
-          <Button
+          <div
+            onClick={toggleShowSelectionOption}
             css={css`
-              margin-left: auto;
+              display: flex;
+              cursor: pointer;
+              align-items: center;
             `}
-            size="sm"
-            onClick={toggleShowSelectionOption}
           >
-            선택된 질문만 보기
-          </Button>
+            <Toggle
+              css={css`
+                margin-left: auto;
+              `}
+              onClick={toggleShowSelectionOption}
+              isToggled={onlySelectedOption}
+            ></Toggle>
```

Comment:
[p-5] 사용되는걸 보니 기분이 좋군요 ㅎㅎ
```suggestion
            <Toggle
              css={css`
                margin-left: auto;
              `}
              onClick={toggleShowSelectionOption}
              isToggled={onlySelectedOption}
            />
```
요렇게 두는게 좀 더 보기 깔끔할 것 같아요~
마찬가지로 요것두 다른 pr에서 반영하셔도 됩니다